internal/pkg/utils/utils.go

```diff
@@ -140,6 +140,30 @@ func ArrayToCommaDelimitedString(arr []string) string {
 	return delimitedStr.String()
 }
 
+func ArrayToCommaDelimitedStringWithAnd(arr []string) string {
```

Comment:
How about passing "and" or "or" into this function?
```suggestion
func SliceToCommaSeparatedString(arr []string, conjunction string) string {
```